### PR TITLE
fix readme with main branch and Neovim typo

### PR DESCRIPTION
**Pull request by eugenemedvediev**
- Consistent Neovim
- changed `master` branch reference to `main`
- `shortmess` setup for lua

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,9 @@ integrations with other projects such as
     version of nvim, v.0.5.x.
 - Ensure [Coursier](https://get-coursier.io/docs/cli-installation) is installed
     on your machine. `nvim-metals` uses Coursier to download and update Metals.
-- Remove `F` from `shortmess`. `set shortmess-=F` _NOTE_: Without doing this,
+- Remove `F` from `shortmess`. `set shortmess-=F`
+    (for lua `vim.o.shortmess = string.gsub(vim.o.shortmess, "F", "") .. "c"`)
```
Comment (ckipp01) [inline README.md:33]: Let's actually use the new syntax for this:

```lua
vim.opt_global.shortmess:remove("F"):append("c")
```

In the future I just might remove all of the non-lua examples.

**Review by ckipp01 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (eugenemedvediev) [reply README.md]: I saw it in `minimal setup` file, but was not sure which is better as I am not strong yet it lua.
Done

Comment (ckipp01) [reply README.md]: 👍🏼 yea that was from before the introduced the new syntax. Thanks for fixing it!

**Review by ckipp01 — APPROVED** [review]
Looks good!

**Comment by ckipp01** [comment]
@mergifyio backport next

**Pull request merged**